client/components/data/query-user-purchases/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
```

Comment:
I checked all existing Query components, and it turns out we never expose React. Interesting. :)
We should skip it here, too. So all good :)